pkgs/applications/version-management/git-and-tools/git-annex-remote-googledrive/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, annexremote
+, drivelib
+, GitPython
+, tenacity
+, humanfriendly
+}:
+
+buildPythonApplication rec {
+  pname = "git-annex-remote-googledrive";
+  version = "1.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1031scjwd5mc2igayg4rbmvb89xdxf1fwkw2101kq3p0grvikp0c";
+  };
+
+  propagatedBuildInputs = [ annexremote drivelib GitPython tenacity humanfriendly ];
+
+  # has no tests
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A git-annex special remote for Google Drive";
+    homepage = "https://pypi.org/project/git-annex-remote-googledrive/";
+    license = licenses.gpl3;
```

Comment:
Is it gpl3Only or gpl3Plus?

Reply:
I thought gpl3 implied that it wasn't an "or later"? Anyhow, changed it.

Reply:
gpl3 is ambiguous and in the way of being deprecated. Please help with that goal by not adding new occurrencs of deprecated licenses.